infra-gen2/tool/deploy_gen2.dart

```diff
@@ -108,15 +124,18 @@ void main(List<String> arguments) async {
       );
     }
 
-    // Copy amplify_outputs.dart to shared paths
-    _copyAmplifyOutputs(
+    // Copy config files to shared paths
+    _copyConfigFile(
       backendGroup.sharedOutputs,
       amplifyOutputs,
     );
-
-    var bucketName = _createBucketName(categoryName);
+    _copyConfigFile(
+      backendGroup.sharedOutputs,
+      amplifyConfiguration,
+    );
```

Comment:
nit: instead of calling the  _copyConfigFile two times we can update the func to accept a list of file instead of one file and so call it once and pass both config files.